<launch>
  <!--  **************** Visualisation ****************  -->
  <!--
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find rbe3002_lab3)/rviz/rviz.rviz"/>

-->
  <!-- New NAvigation nodes -->
  <node name="astar" pkg="rbe3002_lab3" type="astar.py" output="screen" >
    <param name="map_topic" value="/map" />
    <param name="obstacle_expansion" value="true" />
    <param name="service_name" value="astar" />
    <param name="cost_out" value="/closed_nodes" />
  </node>
  <!---
  <node name="local_astar" pkg="rbe3002_lab3" type="astar.py" output="screen" >
    <param name="map_topic" value="/move_base/local_costmap/costmap" />
    <param name="obstacle_expansion" value="false" />
    <param name="service_name" value="local_astar" />
    <param name="cost_out" value="/local_closed_nodes" />
  </node>
  -->
  <node name="run_astar" pkg="rbe3002_lab3" type="run_astar.py" output="screen">
    <param name="goal_set" value="/frontier_path" />
    <param name="go_pub" value="/move_base_simple/goal" />
  </node>

  <node name="frontier" pkg="rbe3002_lab3" type="frontier.py" />
  <!---
  <node name="local_run_astar" pkg="rbe3002_lab3" type="run_astar.py" output="screen" >
    <param name="service_name" value="local_astar" />
    <param name="goal_name" value="/waypoint_global" />
    <param name="path_pub" value="/local_astar_path" />
    <param name="go_pub" value="/waypoint" />
  </node>
  <node name="drive" pkg="rbe3002_lab3" type="drive.py" />
  -->


</launch>
